.buildkite/gcp/pipeline.yml

```diff
@@ -38,13 +33,18 @@ steps:
         command:
 
           - echo "--- Run simulation"
-          - "mpiexec -n 8 julia --threads=4 --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/run_amip.jl --config_file config/longrun_configs/amip_diagedmf_10years.yml --job_id amip_30_h_elem"
+          - |
+            mpiexec -n 8 julia --threads=4 --color=yes \
+                    --project=experiments/ClimaEarth/ \
+                    experiments/ClimaEarth/run_amip.jl \
+                    --config_file config/longrun_configs/amip_diagedmf_10years.yml \
+                    --job_id amip_30_h_elem
         artifact_paths: "experiments/ClimaEarth/output/amip_coarse/artifacts/*"
         env:
           CLIMACOMMS_DEVICE: "CUDA"
           CLIMACOMMS_CONTEXT: "MPI"
         agents:
-          slurm_gpus_per_task: 1
+          slurm_gpus: 8
           slurm_cpus_per_task: 4
```

Comment:
should we change this to `slurm_cpus` too or is it ok?

Reply:
I'm not sure, I can test it. I haven't been able to get `srun` to work either but I think it's ok to use `mpiexec`.

Reply:
This results in `sbatch: option '--cpus=32' is ambiguous; possibilities: '--cpus-per-gpu' '--cpus-per-task'` so I'm going to keep it as `slurm_cpus_per_task`